agent/ansible/collection/roles/pbench_repo_install/defaults/main.yml

```diff
@@ -3,11 +3,19 @@
 # provided by the `ndokos` COPR user.
 fedoraproject_username: ndokos
 pbench_repo_url_prefix: https://copr-be.cloud.fedoraproject.org/results/{{ fedoraproject_username }}
+pbench_aux_repo_name: pbench
 
 repos:
-  - name: "{{ pbench_repo_name }}"
-    user: "{{ fedoraproject_username }}"
+  - tag: "{{ pbench_repo_name }}"
+    user: "{{ fedoraproject_user }}"
```

Comment:
Where is `fedoraproject_user` defined?  (I see a definition for `fedoraproject_username` at line 4....)

Reply:
In my inventory file: I introduced an inconsistency and then propagated it back... Fix coming.